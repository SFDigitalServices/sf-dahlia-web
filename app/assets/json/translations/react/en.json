{
  "config.routePrefix": "",
  "errorBoundary.description": "An application error occurred. Check back later.",
  "errorBoundary.goHome": "Back to homepage",
  "footer.cityCountyOfSf": "City & County of San Francisco",
  "footer.contact": "Contact",
  "footer.dahliaDescription": "DAHLIA: San Francisco Housing Portal is a project of the <br /><a className='text-white' href='%{mohcdUrl}' target='_blank'>Mayor's Office of Housing and Community Development</a>",
  "footer.disclaimer": "Disclaimer",
  "footer.forGeneralQuestions": "For general program inquiries, you may call MOHCD at 415-701-5500.",
  "footer.forListingQuestions": "For listing and application questions, please contact the agent displayed on the listing.",
  "footer.giveFeedback": "Give Feedback",
  "footer.inPartnershipWith": "in partnership with<br /><a className='text-white' href='%{sfdsUrl}' target='_blank'>San Francisco Digital Services</a><br /><a className='text-white' href='%{mayorUrl}' target='_blank'>Mayor's Office of Civic Innovation</a>",
  "footer.privacyPolicy": "Privacy Policy",
  "idleTimeout.sessionExpired": "Your session has expired.",
  "idleTimeout.sessionInactivity": "To protect your identity, your session will expire in one minute due to inactivity. You will lose any unsaved information you've entered into the application for this listing if you choose not to respond.",
  "idleTimeout.sessionInactivityLoggedIn": "To protect your identity, your session will expire in one minute due to inactivity. You will lose any unsaved information and be logged out if you choose not to respond.",
  "idleTimeout.stayLoggedIn": "Stay logged in?",
  "label.clear": "Clear",
  "label.editEligbility": "Edit Eligibility",
  "languages.en": "English",
  "languages.es": "Español",
  "languages.tl": "Filipino",
  "languages.zh": "中文",
  "listingDetails.applicationDeadline.open": "Application Deadline %{date} at %{time}",
  "listingDetails.applicationDeadline.closed": "Applications Closed %{date} at %{time}",
  "listings.additional.hide": "Hide additional listings",
  "listings.additional.show": "Show additional listings",
  "listings.additional.subtitle": "We know you may have options about how many people will live with you. Here are listings for other household sizes and income levels.",
  "listings.additional.title": "Additional Listings",
  "listings.additionalInformation.header": "Additional Information",
  "listings.additionalInformation.subheader": "Required documents and selection criteria",
  "listings.applicationDeadline": "Application Deadline",
  "listings.applicationsClosed": "Applications Closed",
  "listings.atTotalIncome": "at <span class='eligibility-subtext'>$%{income}</span> %{per}",
  "listings.availableUnits": "Available Units",
  "listings.availableUnitsAndOpenWaitlist": "Available Units & Open Waitlist",
  "listings.buildingImageAltText": "A picture of the building",
  "listings.cc&r": "Covenants, Conditions and Restrictions (CC&R's)",
  "listings.cc&r_description": "The CC&R's explain the rules of the homeowners' association, and restrict how you can modify the property.",
  "listings.download_pdf": "Download PDF",
  "listings.eligibility.header": "Eligibility",
  "listings.eligibility.subheader": "Income, occupancy, preferences, and subsidies",
  "listings.eligibilityCalculator.matched": "Matched",
  "listings.eligibilityCalculator.notAMatch": "Not a Match",
  "listings.eligibilityCalculator.rent.noMatchingUnits": "Based on information you entered, you don't match any current listings for rent.",
  "listings.eligibilityCalculator.rent.showingMatchingUnits": "Showing matching units for rent",
  "listings.eligibilityCalculator.sale.noMatchingUnits": "Based on information you entered, you don't match any current listings for sale.",
  "listings.eligibilityCalculator.sale.showingMatchingUnits": "Showing matching units for sale",
  "listings.eligibilityCalculator.youMayBeEligible": "Based on information you entered, you may be eligible for units at the following property.",
  "listings.equal_housing_opportunity_logo": "Equal Housing Opportunity Logo",
  "listings.features.header": "Features",
  "listings.features.rentSubheader": "Amenities, unit details and additional fees",
  "listings.features.saleSubheader": "Amenities and unit details",
  "listings.forHouseholdSize": "for <span class='eligibility-subtext'>%{size}</span> %{people}",
  "listings.habitat.infoSession": "You will need to attend an information session.",
  "listings.habitat.payments": "Payments",
  "listings.habitat.payments.desc": "No down payment, but you do need to complete 500 hours of sweat equity. Your monthly payment will be 30% of your income.",
  "listings.important_program_rules": "Important Program Rules",
  "listings.includesPriorityUnits": "Includes Priority Units for %{priorities}",
  "listings.includingChildren": "(including <span class='eligibility-subtext'>%{number} %{children}</span> under 6)",
  "listings.lotteryResults.cardTitle": "Lottery Results Posted",
  "listings.lotteryResults.hide": "Hide Lottery Results",
  "listings.lotteryResults.noResults": "There are no listings with lottery results at this time.",
  "listings.lotteryResults.show": "Show Lottery Results",
  "listings.lotteryResults.subtitle": "You can now check lottery results on the remaining listings.",
  "listings.lotteryResults.title": "Lottery Results",
  "listings.monitored_by_mohcd": "Monitored by the Mayor's Office of Housing & Community Development",
  "listings.neighborhood.header": "Neighborhood",
  "listings.neighborhood.subheader": "Location and transportation",
  "listings.noMatches": "No Matches",
  "listings.noOpenListings": "No listings currently have open applications.",
  "listings.openWaitlist": "Open Waitlist",
  "listings.people": "people",
  "listings.person": "person",
  "listings.prioritiesDescriptor.hearing": "Hearing Impairments",
  "listings.prioritiesDescriptor.hearingVision": "Vision and/or Hearing Impairments",
  "listings.prioritiesDescriptor.mobility": "Mobility Impairments",
  "listings.prioritiesDescriptor.mobilityHearingVision": "Mobility, Hearing and/or Vision Impairments",
  "listings.prioritiesDescriptor.vision": "Vision Impairments",
  "listings.process.header": "Process",
  "listings.process.subheader": "Important dates and contact information",
  "listings.realtor_commission": "For the Buyer's Realtor",
  "listings.realtor_commission_header": "Your Commission: ",
  "listings.realtor_commission_how_to": "How to receive it: ",
  "listings.realtor_commission_not_eligible": "Your realtor is not eligible for a commission on this unit",
  "listings.re_pricing": "Resale Price Restrictions",
  "listings.required_documents": "Required Documents",
  "listings.reservedCommunityType.accessible": "Accessible Units Only",
  "listings.reservedCommunityType.artist": "Artist Loft Building",
  "listings.reservedCommunityType.disability": "Developmental Disability Building",
  "listings.reservedCommunityType.habitat": "Habitat Greater San Francisco",
  "listings.reservedCommunityType.senior": "Senior Building",
  "listings.reservedCommunityType.veteran": "Veterans Building",
  "listings.unitTypes.1 BR": "1 Bedroom",
  "listings.unitTypes.2 BR": "2 Bedroom",
  "listings.unitTypes.3 BR": "3 Bedroom",
  "listings.unitTypes.4 BR": "4 Bedroom",
  "listings.unitTypes.5 BR": "5 Bedroom",
  "listings.unitTypes.SRO": "SRO",
  "listings.unitTypes.Studio": "Studio",
  "listings.upcomingLotteries.hide": "Hide Upcoming Lotteries",
  "listings.upcomingLotteries.noResults": "There are no closed listings with upcoming lotteries at this time.",
  "listings.upcomingLotteries.show": "Show Upcoming Lotteries",
  "listings.upcomingLotteries.subtitle": "The application deadline has passed on the following listings, and you can still get information on the lottery date and time.",
  "listings.upcomingLotteries.title": "Upcoming Lotteries",
  "listings.special_notes": "Special Notes",
  "listings.waitlist.open": "Open Waitlist",
  "listings.weSuggestHousingCounselor": "For more help, we suggest talking with a housing counselor to explore your options. <a href='%{url}' target='_blank'>Find a housing counselor</a>.",
  "nav.accountSettings": "Account Settings",
  "nav.buy": "Buy",
  "nav.getAssistance": "Get Assistance",
  "nav.getFeedback": "We'd love to get <a target='_blank' href='%{feedbackUrl}'>your feedback.</a>",
  "nav.myAccount": "My Account",
  "nav.myApplications": "My Applications",
  "nav.myDashboard": "My Dashboard",
  "nav.myFavorites": "My Favorites",
  "nav.rent": "Rent",
  "nav.researchFeedback": "<a target='_blank' href='%{researchUrl}'>Sign up</a> to provide feedback on City websites",
  "nav.signIn": "Sign In",
  "nav.signOut": "Sign Out",
  "pageTitle.rentalListings": "Rental Listings",
  "pageTitle.saleListings": "Sale Listings",
  "pageTitle.signIn": "Sign In",
  "rentalDirectory.calloutbutton": "View opportunities",
  "rentalDirectory.callouttitle": "See more listings from our community partners.",
  "rentalDirectory.findMatchingListings": "Find matching listings",
  "rentalDirectory.ifYouTellUs": "If you tell us your household size and income, we can show you listings you may qualify for.",
  "rentalDirectory.orGetHelpCalculating": "or <a href='%{incomeLink}'>get help calculating your income</a>",
  "rentalDirectory.title": "Rent affordable housing",
  "saleDirectory.beforeApplying.makeSureYou": "Make sure you:",
  "saleDirectory.beforeApplying.readFullList": "Read <a href='%{url}' target='_blank'>our full list of requirements</a> for more details.",
  "saleDirectory.beforeApplying.step1": "Haven't owned residential property in the past 3 years",
  "saleDirectory.beforeApplying.step2": "Meet our <a href='%{url}' target='_blank'>income requirements</a>",
  "saleDirectory.beforeApplying.step3": "Complete <a href='%{url}' target='_blank'>homebuyer education</a>",
  "saleDirectory.beforeApplying.step4": "Get pre-approved for a mortgage loan by a <a href='%{url}' target='_blank'>lender on our list</a>",
  "saleDirectory.beforeApplying.step5": "Have enough in savings for closing costs and downpayment",
  "saleDirectory.beforeApplying.title": "Before applying",
  "saleDirectory.callout.citySecondLoan": "City Second Loan Program",
  "saleDirectory.callout.firstComeFirstServed": "First come, first served",
  "saleDirectory.callout.title": "Find more listings on our website:",
  "saleDirectory.getHelp.dalpProgramInfo": "The 2021 Downpayment Assistance Loan Program (DALP) will begin accepting applications on February 26, 2021.",
  "saleDirectory.getHelp.helpWithDownpayment": "Help with downpayments",
  "saleDirectory.getHelp.learnMoreAndHowToApply": "Learn more about DALP and how to apply on SF.gov",
  "saleDirectory.getHelp.mohcdPrograms": "The Mayor's Office of Housing and Community Development (MOHCD) offers a number of programs to help homebuyers in San Francisco.",
  "saleDirectory.getHelp.moreResources": "More resources",
  "saleDirectory.getHelp.otherGroupsMayQualify": "Others, like San Francisco teachers and first responders, may qualify for loans.",
  "saleDirectory.getHelp.somePeopleMayQualify": "Some people may qualify for downpayment assistance.",
  "saleDirectory.getHelp.title": "Get help",
  "saleDirectory.getHelp.viewFullListOfPrograms": "View the full list of programs",
  "saleDirectory.getHelp.visitMohcdPrograms": "Visit the MOHCD Affordable Homebuyer Programs page to learn more.",
  "saleDirectory.hoaDues": "HOA Dues",
  "saleDirectory.price": "Sales Price",
  "saleDirectory.seeTheListings": "See the listings",
  "saleDirectory.title": "Buy affordable housing",
  "signIn.badCredentials": "Invalid login credentials. Please try again.",
  "signIn.forgotPassword": "Forgot password?",
  "signIn.signedOutSuccessfully": "You have successfully logged out of your account.",
  "signIn.userTokenValidationTimeout": "We care about your security. We logged you out due to a connection issue. Please sign in to continue.",
  "t.available": "available",
  "t.child": "child",
  "t.children": "children",
  "t.close": "Close",
  "t.continue": "Continue",
  "t.dahliaSanFranciscoHousingPortal": "DAHLIA San Francisco Housing Portal",
  "t.homePage": "Home page",
  "t.income": "income",
  "t.incomeRange": "Income Range",
  "t.menu": "Menu",
  "t.minimumIncome": "Minimum Income",
  "t.numberRange": "%{minValue} to %{maxValue}",
  "t.perMonth": "per month",
  "t.perYear": "per year",
  "t.rent": "Rent",
  "t.seeDetails": "See Details",
  "t.skipToMainContent": "Skip to main content",
  "t.to": "to",
  "t.unitType": "Unit Type",
  "t.units": "Units",
  "t.waitlist": "Waitlist",
  "welcome.newListingEmailAlert": "Get emailed whenever a new listing is posted.",
  "welcome.seeRentalListings": "Rent",
  "welcome.seeSaleListings": "Buy",
  "welcome.signUpToday": "Sign Up today",
  "welcome.title": "Apply for affordable housing"
}
