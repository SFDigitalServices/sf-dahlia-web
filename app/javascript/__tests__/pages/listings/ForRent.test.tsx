import React from "react"
import { render, cleanup, screen, waitFor } from "@testing-library/react"
import ForRent from "../../../../javascript/pages/listings/for-rent"
import { sroRentalListing } from "../../data/RailsRentalListing/listing-rental-sro"

const axios = require("axios")

jest.mock("axios")

describe("For Rent", () => {
  afterEach(() => {
    cleanup()
    jest.clearAllMocks()
    jest.resetAllMocks()
  })

  it("renders ForRent component", async (done) => {
    axios.get.mockResolvedValue({ data: { listings: [] } })

    const { findByText, asFragment } = render(<ForRent assetPaths="/" />)

    expect(await findByText("Rent affordable housing")).toBeDefined()
    expect(asFragment()).toMatchSnapshot()
    done()
  })

  it("listings with multiple listings render the first image in the array", async (done) => {
    axios.get.mockResolvedValue({ data: { listings: [sroRentalListing] } })

    const { findByAltText } = render(<ForRent assetPaths="/" />)

    await waitFor(
      () => {
        return screen
          .getByRole("button", {
            name: /show upcoming lotteries \(1\)/i,
          })
          .click()
      },
      {
        timeout: 10000,
      }
    )

    const image = await findByAltText(`${sroRentalListing.Building_Name} Building`)
    expect(image.getAttribute("src")).toBe(sroRentalListing.Listing_Images[0].displayImageURL)
    done()
  })
})
