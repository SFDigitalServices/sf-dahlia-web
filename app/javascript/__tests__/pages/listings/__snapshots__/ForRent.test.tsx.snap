// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`For Rent renders ForRent component 1`] = `
<DocumentFragment>
  <div
    class="notranslate site-wrapper"
  >
    <div
      class="site-content"
    >
      <title>
        Rental Listings
      </title>
      <meta
        charset="utf-8"
      />
      <meta
        content="IE=edge"
        http-equiv="X-UA-Compatible"
      />
      <meta
        content="width=device-width, initial-scale=1"
        name="viewport"
      />
      <meta
        content="M1zcm6GGM6sHSF_jvkq254DbYAj94JYbFC7ArZDAXlg"
        name="google-site-verification"
      />
      <meta
        content="Rental Listings"
        property="og:title"
      />
      <meta
        property="og:image"
      />
      <meta
        content="website"
        property="og:type"
      />
      <link
        rel="icon"
        sizes="32x32"
        type="image/png"
      />
      <link
        rel="icon"
        sizes="96x96"
        type="image/png"
      />
      <link
        rel="icon"
        sizes="16x16"
        type="image/png"
      />
      <link
        rel="apple-touch-icon"
        sizes="57x57"
      />
      <link
        rel="apple-touch-icon"
        sizes="60x60"
      />
      <link
        rel="apple-touch-icon"
        sizes="72x72"
      />
      <link
        rel="apple-touch-icon"
        sizes="76x76"
      />
      <link
        rel="apple-touch-icon"
        sizes="114x114"
      />
      <link
        rel="apple-touch-icon"
        sizes="120x120"
      />
      <link
        rel="apple-touch-icon"
        sizes="144x144"
      />
      <link
        rel="apple-touch-icon"
        sizes="152x152"
      />
      <link
        rel="apple-touch-icon"
        sizes="180x180"
      />
      <link
        rel="icon"
        sizes="192x192"
        type="image/png"
      />
      <link
        rel="manifest"
      />
      <meta
        name="msapplication-TileImage"
      />
      <header
        class="site-header"
      >
        <a
          class="site-header__skip-link"
          href="#main-content"
        >
          Skip to main content
        </a>
        <div
          class="language-bar"
        >
          <div
            class="language-bar__inner"
          >
            <nav
              aria-label="Choose your language"
              class="language-nav"
            >
              <ul
                class="language-nav__list"
              >
                <li>
                  <button
                    class="language-nav__list-button is-active"
                    type="button"
                  >
                    English
                  </button>
                </li>
                <li>
                  <button
                    class="language-nav__list-button"
                    type="button"
                  >
                    Español
                  </button>
                </li>
                <li>
                  <button
                    class="language-nav__list-button"
                    type="button"
                  >
                    中文
                  </button>
                </li>
                <li>
                  <button
                    class="language-nav__list-button"
                    type="button"
                  >
                    Filipino
                  </button>
                </li>
              </ul>
            </nav>
          </div>
        </div>
        <div
          class="site-header__notice "
        >
          <div
            class="site-header__notice-text"
          >
            <span>
              We'd love to get 
              <a
                href="https://airtable.com/shrw64DubWTQfRkdo"
                target="_blank"
              >
                your feedback.
              </a>
            </span>
          </div>
        </div>
        <nav
          aria-label="main navigation"
          class="site-header__container"
          role="navigation"
        >
          <div
            class="site-header__base "
          >
            <div
              class="site-header__logo-container site-header__logo-width-med"
            >
              <a
                aria-label="DAHLIA San Francisco Housing Portal"
                class="site-header__logo translate site-header__custom-width"
                href="/"
                target="_self"
              >
                <div
                  class="site-header__logo-content site-header__image-only-container"
                >
                  <img
                    alt="Site logo"
                    class="site-header__logo-image site-header__image-only"
                  />
                </div>
              </a>
            </div>
            <div
              class="site-header__navbar-menu"
            >
              <a
                class="site-header__link pb-4 pt-1 flex items-end "
                data-testid="Rent-0"
                href="/listings/for-rent"
                target="_self"
              >
                <div>
                  Rent
                </div>
              </a>
              <a
                class="site-header__link pb-4 pt-1 flex items-end "
                data-testid="Buy-1"
                href="/listings/for-sale"
                target="_self"
              >
                <div>
                  Buy
                </div>
              </a>
              <a
                class="site-header__link pb-4 pt-1 flex items-end "
                data-testid="My Favorites-2"
                href="/favorites"
                target="_self"
              >
                <div>
                  My Favorites
                </div>
              </a>
              <a
                class="site-header__link pb-4 pt-1 flex items-end "
                data-testid="Get Assistance-3"
                href="/get-assistance"
                target="_self"
              >
                <div>
                  Get Assistance
                </div>
              </a>
              <a
                class="site-header__link pb-4 pt-1 flex items-end "
                data-testid="Sign In-4"
                href="/sign-in"
                target="_self"
              >
                <div>
                  Sign In
                </div>
              </a>
            </div>
          </div>
        </nav>
      </header>
      <main
        class="md:overflow-x-hidden"
        data-testid="main-content-test-id"
        id="main-content"
      >
        <div
          aria-live="assertive"
          role="alert"
        >
          <div>
            <header
              class="page-header"
            >
              <hgroup
                class="page-header__group"
              >
                <h1
                  class="page-header__title"
                  data-testid="page-header"
                >
                  Rent affordable housing
                </h1>
                <p
                  class="page-header__lead"
                >
                   If you tell us your household size and income, we can show you listings you may qualify for.
                </p>
                <p
                  class="mt-4 md:mt-8 mb-2"
                >
                  <a
                    class="button"
                    href="/eligibility-estimator/rental"
                    tabindex="0"
                    target="_self"
                  >
                    Find matching listings
                  </a>
                </p>
                <a
                  class="text-base text-primary-dark"
                  href="/income-calculator/rental/intro"
                >
                  or get help calculating your income
                </a>
              </hgroup>
            </header>
            <div
              id="listing-results"
            >
              <div
                class="bg-primary-darker"
              >
                <div
                  class="max-w-5xl mx-auto p-2 md:p-4"
                >
                  <div
                    class="action-block  primary-darker action-block__inline"
                  >
                    <div
                      class="action-block__head"
                    >
                      <div
                        class="action-block__header"
                      >
                        <h2
                          class=""
                        >
                          See more listings from our community partners.
                        </h2>
                      </div>
                    </div>
                    <div
                      class="action-block__actions"
                    >
                      <a
                        class="button"
                        href="/additional-resources"
                        target="_self"
                      >
                        View opportunities
                      </a>
                    </div>
                  </div>
                </div>
              </div>
            </div>
            <div
              class="action-block mt-4 primary-lighter action-block__block"
            >
              <div
                class="action-block__head"
              >
                <div
                  class="action-block__icon"
                >
                  <span
                    class="ui-icon ui-3xl"
                  >
                    <svg
                      fill="transparent"
                      height="35"
                      viewBox="0 0 80 35"
                      width="80"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <rect
                        fill="transparent"
                        height="75"
                        width="80"
                      />
                      <g
                        clip-path="url(#clip0)"
                      >
                        <rect
                          fill="transparent"
                          height="1390"
                          transform="translate(-284 -530)"
                          width="1280"
                        />
                        <path
                          d="M61.3749 24.2402L43.2061 39.7424C41.8262 40.8341 39.7563 40.8341 38.3764 39.7424L19.9775 24.2402"
                          stroke="black"
                          stroke-linecap="round"
                          stroke-linejoin="round"
                          stroke-miterlimit="10"
                          stroke-width="2"
                        />
                        <path
                          d="M49.1858 41.271L61.375 50.878"
                          stroke="black"
                          stroke-linecap="round"
                          stroke-linejoin="round"
                          stroke-miterlimit="10"
                          stroke-width="2"
                        />
                        <path
                          d="M32.1668 41.271L19.9775 50.878"
                          stroke="black"
                          stroke-linecap="round"
                          stroke-linejoin="round"
                          stroke-miterlimit="10"
                          stroke-width="2"
                        />
                        <path
                          d="M14.458 24.0218V51.0961C14.458 53.9345 16.7579 56.1179 19.7477 56.1179H61.835C64.8248 56.1179 67.1247 53.9345 67.1247 51.0961V24.0218C67.1247 21.1834 64.8248 19 61.835 19H19.7477C16.7579 19 14.458 21.1834 14.458 24.0218Z"
                          stroke="black"
                          stroke-linecap="round"
                          stroke-linejoin="round"
                          stroke-miterlimit="10"
                          stroke-width="2"
                        />
                      </g>
                      <defs>
                        <clippath
                          id="clip0"
                        >
                          <rect
                            fill="white"
                            height="1390"
                            transform="translate(-284 -530)"
                            width="1280"
                          />
                        </clippath>
                      </defs>
                    </svg>
                  </span>
                </div>
                <div
                  class="action-block__header"
                >
                  <h2
                    class=""
                  >
                    Get emailed whenever a new listing is posted.
                  </h2>
                </div>
              </div>
              <div
                class="action-block__actions"
              >
                <a
                  class="button"
                  href="https://confirmsubscription.com/h/y/C3BAFCD742D47910"
                  target="_blank"
                >
                  Sign Up today
                </a>
              </div>
            </div>
          </div>
        </div>
      </main>
    </div>
    <footer
      class="site-footer"
    >
      <div
        class="footer-row "
      >
        <div
          class="footer-row__section pb-8"
        >
          <img
            alt=""
            data-testid="footer-logo-test-id"
          />
        </div>
      </div>
      <div
        class="footer-row "
      >
        <div
          class="footer-row__section py-2"
        >
          <p
            class="text-gray-500"
          >
            <span>
              DAHLIA: San Francisco Housing Portal is a project of the 
              <br />
              <a
                class="text-white"
                href="https://sf.gov/mohcd"
                target="_blank"
              >
                Mayor's Office of Housing and Community Development
              </a>
            </span>
          </p>
          <p
            class="text-xs mt-4 text-gray-500"
          >
            <span>
              in partnership with
              <br />
              <a
                class="text-white"
                href="https://digitalservices.sfgov.org/"
                target="_blank"
              >
                San Francisco Digital Services
              </a>
              <br />
              <a
                class="text-white"
                href="https://www.innovation.sfgov.org/"
                target="_blank"
              >
                Mayor's Office of Civic Innovation
              </a>
            </span>
          </p>
        </div>
      </div>
      <div
        class="footer-row "
      >
        <div
          class="footer-row__section pb-8"
        >
          <p
            class="text-sm"
          >
            For listing and application questions, please contact the agent displayed on the listing. 
            <br />
            For general program inquiries, you may call MOHCD at 415-701-5500.
          </p>
        </div>
      </div>
      <section
        class="footer-sock"
      >
        <div
          class="footer-sock__inner"
        >
          <p
            class="footer-copyright"
          >
            © City & County of San Francisco
          </p>
          <nav
            aria-label="Footer"
            class="footer-nav"
          >
            <a
              class="text-gray-500"
              href="https://airtable.com/shrw64DubWTQfRkdo"
              target="_blank"
            >
              Give Feedback
            </a>
            <a
              class="text-gray-500"
              href="mailto:sfhousinginfo@sfgov.org"
              target="_self"
            >
              Contact
            </a>
            <a
              class="text-gray-500"
              href="/disclaimer"
              target="_self"
            >
              Disclaimer
            </a>
            <a
              class="text-gray-500"
              href="/privacy"
              target="_self"
            >
              Privacy Policy
            </a>
          </nav>
        </div>
      </section>
    </footer>
  </div>
</DocumentFragment>
`;
